File: Solutions/921/921.java
class Solution {
    public int minAddToMakeValid(String s) {
        int count = 0;
        int missed = 0;
        for (char x : s.toCharArray()) {
            if (x == '(') {
                count++;
            } else {
                if (count > 0) {
                    count--;
                } else {
                    missed++;
                }
            }
        }
        return count+missed;
    }
}

File: Solutions/921/921.py
class Solution(object):
    def minAddToMakeValid(self, s):
        """
        :type s: str
        :rtype: int
        """
        count = 0
        missed = 0
        for x in s:
            if x == "(":
                count += 1
            else:
                if count > 0:
                    count -= 1
                else:
                    missed += 1
        return count+missed
